src/architecture/SF2.0-service-instance-bosh-provisioning.seq

```diff
@@ -0,0 +1,60 @@
+seqdiag {
+  # Change edge metrics
+  edge_length = 150; // default value is 192
+  span_height = 0;   // default value is 40
+
+  # Change note color
+  default_note_color = lightblue;
+
+  # actors (in order left to right)
+  User; Controller; Broker; APIServer; BOSHManager; Bosh; Service; IaaS;
+
+  # trigger provisioning of service instance
+  User ->> Controller [label = "cf create-service"] {
+    Controller => Broker [label = "run provisioning", return = "ok"] {
+      Broker => APIServer [label = "create lock resource on deployment", return = "true"];
+      Broker => APIServer [label = "Create a director resource with state as in_queue", return = "true"];
+    }
+  }
+
+  APIServer -> BOSHManager [label = "Trigger watcher for new create request"]
+  BOSHManager ->> Bosh [label = "bosh deploy"]
+  BOSHManager -> APIServer [label = "sets resource state as in_progress", return = "true"];
```

Comment:
maybe you can mention `director resource` here.

Reply:
cool!